napalm_logs/config/fortinet/LOCAL_TRAFFIC.yml

```diff
@@ -0,0 +1,41 @@
+# Logs files for traffic identified as type=traffic and subtype=local (FortiOs 5.4).
+#
+# <189>date=2019-04-04 time=09:19:21 devname=fw01 devid=FG800D0123456789 logid=0001000014 type=traffic 
+# subtype=local level=notice vd=root srcip=172.30.0.1 srcport=48770 srcintf="wan1" dstip=172.16.0.1 
+# dstport=443 dstintf="root" sessionid=3841389285 proto=6 action=close policyid=0 policytype=local-in-policy 
+# dstcountry="France" srccountry="France" trandisp=noop service="HTTPS" app="Web Management(HTTPS)" 
+# duration=3 sentbyte=28187 rcvdbyte=334857 sentpkt=183 rcvdpkt=242 appcat="unscanned"
+#
+messages:
+  - error: 'LOCAL_TRAFFIC'
+    tag: "local"
+    values:
+      level: (?<=level=)(.*)(?=\s+vd=)
```

Comment:
Because the `line` contains `level={level} vd={vd}...`, which once these values are substituted, becomes `level=(?<=level=)(.*)(?=\s+vd=) vd=(?<=vd=)(.*)(?=\s+srcip=)...`, I see no reason to have the lookahead and lookbehind as `level=(.*) vd=(.*)...` would achieve the same. However as we know `level` will always be a word character I would recommend using `(\w+)`.

Reply:
you are right.